pages/report-repair/[route].js

```diff
@@ -100,6 +100,10 @@ function ReportRepair() {
     })
   }
 
+  const commonProblems = {
+    walls: { value: 'wallsFloorsCeiling', title: 'Walls, floor or ceiling, excluding damp' }
```

Comment:
The current name of `walls` could be misleading when taken out of context and assumed to only be for "walls", whereas it includes floor and ceiling too.
Please rename to represent all the areas it encompasses.
```suggestion
    wallsFloorAndCeiling: { value: 'wallsFloorsCeiling', title: 'Walls, floor or ceiling, excluding damp' }
```